business/istio_config.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 	"sync"
 
+	"gopkg.in/yaml.v2"
```

Comment:
I just see many places we use different yaml packages to deal with yamls.. 😄

Reply:
This package is also used in https://github.com/kiali/kiali/blob/master/business/mesh.go#L11 or https://github.com/kiali/kiali/blob/master/handlers/config.go#L9, not sure what other packages are used but, I was thinking that maybe that can be addressed in a separate issue?

Reply:
AFAIK, we almost exclusively use this gopkg.in... I don't know where we use anything else. However, I can say we only recently are about to add the k8s yaml utility in one place, but that is only because it we can't use this gopkg.in yaml package -- only the k8s yaml was able to get the yaml to demarshal into the auto-generated protobuf code (that was done recently [here](https://github.com/kiali/kiali/pull/6027/files#diff-445738cf30752e18544dc219fe6aaeb2f7b10bc9be22a38a2ed5089783f05dc3R681)).

Reply:
That is right, I saw this [change](https://github.com/kiali/kiali/pull/6027/files#diff-445738cf30752e18544dc219fe6aaeb2f7b10bc9be22a38a2ed5089783f05dc3R681) recently, so I point this out, no blocker on this.